main.py

```diff
@@ -0,0 +1,58 @@
+import sqlalchemy
+
+
+db = 'postgresql://postgres:admin@localhost:5432/postgres'
+engine = sqlalchemy.create_engine(db)
+connection = engine.connect()
+
+# print(engine.get_table_names())
+# """What tables are in the database"""
+
+# name and year of release of albums released in 2018
+select_1 = connection.execute(
```

Comment:
It is better to use additional variable to store query without editing execute line.